### PR TITLE
Integrate Roman clothing & headgear from Silfae

### PR DESCRIPTION
**Pull request by rquinio**
I've integrated the contribution from Silfae and also re-factored things a bit to be compatible with Iberian Portraits DLC. 
Could you please review the list of cultures that should get the romangfx ethnicity ?

Changes:
- Rename assets from the old Mediterranean Portraits DLC from roman to mediterranean
- Define new ethnicity romangfx combining mediterranean faces and Silfae's roman clothings/headgear
- Set romangfx for cultures from western roman empire (roman, romano_gallic, romano_hispanic, illyro_roman, thraco_roman, latin, corsican, dalmatian, pannonian, maltanesse)
- Split portraits definition for mediterraneangfx, andalusiangfx and romangfx in separate files
- Use mediterraneangfx as fallback for southerngfx, if not having Iberian Portraits DLC.
- Move WTWSMS graphicalculturetypes to separate file
- Tweak properties for romangfx to have higher chances of not using beards
- Override merchants portraits to use Silfae's early clothings/ no headgear for republics

### CODE REVIEW

File: WTWSMS/interface/portrait_properties.txt [hunk 1]
```diff
@@ -0,0 +1,1389 @@
...
+			factor = 80
+			OR = {
+				culture = roman
+				culture = romano_gallic
+				culture = romano_hispanic
+				culture = thraco_roman
+				culture = illyro_roman
+				culture = latin
+			}
+			graphical_culture = romangfx
```
Comment (rquinio) [inline WTWSMS/interface/portrait_properties.txt]: Here's the addition from vanilla portrait_properties.
It makes much more likely that Romans will not grow beards, as the Mediterranean long beards don't feel very Roman to me (but maybe that's just me :p)

File: WTWSMS/interface/portrait_properties.txt [hunk 2]
```diff
@@ -0,0 +1,1389 @@
...
+			factor = 5
+			OR = {
+				culture = roman
+				culture = romano_gallic
+				culture = romano_hispanic
+				culture = thraco_roman
+				culture = illyro_roman
+				culture = latin
+			}
+			graphical_culture = romangfx
```
Comment (rquinio) [inline WTWSMS/interface/portrait_properties.txt]: Minor addition here too, index 9 is the smallest Mediterranean beard.

**Comment by loup99** [comment]
The cultures you have defined for the moment look good, but I'm personally not an expert, and a lot of the other cultures we have either never appear ingame or are kind of obsolete. I don't know really, hopefully someone else in the team (Erik W for instance) with more knowledge on this can clarify things.

**Comment by loup99** [comment]
With the review accepted, I'm trying to merge the two branches. So far unsucessful.

**Pull request merged**